packages/nx-monorepo/src/utils/node.ts

```diff
@@ -19,6 +24,14 @@ export namespace NodePackageUtils {
     return cmd;
   }
 
+  /** Indicates if project is a node based project */
+  export function isNodeProject(project: Project): boolean {
+    if (project instanceof NodeProject) {
```

Comment:
Can be simplified to:

`return project instanceof NodeProject || tryFindNodePackage(project, false) != null;`